### PR TITLE
Начинаем разметку личного проекта

### PR DESCRIPTION
**Pull request by TsoriTheBoss**
Разметил главную.
Пока без всплывающих окон, их добавлю позже.

---
:mortar_board: [Начинаем разметку личного проекта](https://up.htmlacademy.ru/htmlcss/20/user/570231/tasks/13)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,252 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
+
+<head>
     <meta charset="utf-8">
```
[the remark this answers is not in the document]
Comment (TsoriTheBoss) [reply index.html]: I will add a meta tags when will connect the files of the css, fonts, viewport.

File: index.html [hunk 2]
```diff
@@ -1,12 +1,252 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
+
```
[the remark this answers is not in the document]
Comment (TsoriTheBoss) [reply index.html]: Empty lines are left temporarily for convenience of search and addition of additional blocks at css style

File: index.html [hunk 3]
```diff
@@ -1,252 +1,346 @@
...
+                    <div class="dropped" style="display: none">
+                        <form class="header__login-dropped" method="post" action="https://echo.htmlacademy.ru">
+                            <label for="header__login-email-field"></label>
+                            <input id="header__login-email-field" type="email" name="header__login-email-field" value="" placeholder="Электронная почта">
+                            <label for="header__login-password-field"></label>
+                            <input id="header__login-password-field" type="password" name="header__login-password-field" value="" placeholder="Пароль">
+                            <input id="header__login-btn" type="submit" value="Войти">
+                            <div class="header__login-links">
+                                <a class="header__login-link" href="#">Забыли пароль?</a>
+                                <a class="header__login-link" href="#">Новая регистрация</a>
```
Comment (mihail910) [inline index.html]: Add button for this form as i wrote abive

File: index.html [hunk 4]
```diff
@@ -1,252 +1,346 @@
...
+                        </form>
+                    </div>
+                </div>
+                <div class="header__basket">
+                    <a class="header__basket-link" href="#">
+                        <img class="header__login-picture" src="img/basket-empty.svg" alt="Корзина">
+                        <p class="header__login-description">Пусто</p>
+                    </a>
+                    <div class="dropped" style="display: none">
+                        <span>Выпадающий список товаров в корзине</span>
```
Comment (mihail910) [inline index.html]: It's better to use DIV. because every basket item shouldn't be INLINE. It's really block

File: index.html [hunk 5]
```diff
@@ -1,252 +1,346 @@
...
-                <img src="img/slider-slide-1.png" alt="Глейси">
+        <main class="container">
 
-                <p class="text-switching">
-                    Крем-брюле и<br> пломбир с малиновым<br> джемом
-                </p>
+            <div class="main__slider">
 
-                <a class="btn-index-in-slider" href="#">Давайте оба!</a>
+                <input class="visually-hidden" id="first" class="btn-switching" type="radio" checked>
```
Comment (mihail910) [inline index.html]: bind naming with slider. like slider__input-switcher

File: index.html [hunk 6]
```diff
@@ -1,252 +1,346 @@
...
-                <input class="btn-switching" type="radio">
-                <input class="btn-switching" type="radio">
-                
-                <img src="img/slider-slide-1.png" alt="Глейси">
+        <main class="container">
 
-                <p class="text-switching">
-                    Крем-брюле и<br> пломбир с малиновым<br> джемом
-                </p>
+            <div class="main__slider">
```
Comment (mihail910) [inline index.html]: It's not main slider. It's simple SLIDER

File: index.html [hunk 7]
```diff
@@ -1,252 +1,346 @@
...
 
-                <a class="btn-index-in-slider" href="#">Давайте оба!</a>
+                <input class="visually-hidden" id="first" class="btn-switching" type="radio" checked>
+                <label class="main__slider-switcher" for="first"></label>
```
Comment (mihail910) [inline index.html]: remove MAIN_slider. Let it be 'slider'

File: index.html [hunk 8]
```diff
@@ -1,252 +1,346 @@
...
-                <p class="index-promo-text">При покупке 2 кг любого фруктового мороженого, добавим в ваш заказ банку малинового варенья бесплатно.</p>
-                <a class="btn-index-promo" href="#">Хочу варенье!</a>
-            </section>
-
-            <section class="index-promo-right">
-                <h2 class="visually-hidden">Акция</h2>
-                <h3 class="index-promo-caption">Шоколадки даром!</h3>
-                <p class="index-promo-text">При покупке 2 кг пломбира, добавим в ваш заказ упаковку вкуснейшей шоколадной присыпки совершенно бесплатно.</p>
-                <a class="btn-index-promo" href="#">Хочу шоколадки!</a>
+            <section class="main__promo">
```
Comment (mihail910) [inline index.html]: get rid of 'main'. it's promo. can be used in different pages

File: index.html [hunk 9]
```diff
@@ -1,252 +1,346 @@
...
+                <div class="main__promo-item">
+                    <h2 class="visually-hidden">Акция</h2>
+                    <h3 class="main__promo-title">Шоколадки даром!</h3>
+                    <p class="main__promo-description">При покупке 2 кг пломбира, добавим в ваш заказ упаковку вкуснейшей шоколадной присыпки совершенно бесплатно.</p>
+                    <a class="main__promo-btn" href="#">Хочу шоколадки!</a>
+                </div>
             </section>
 
-            <section class="index-popular-items">
+            <section class="main__popular-items">
```
Comment (mihail910) [inline index.html]: as i wrote abive. get rid of main. and further

File: index.html [hunk 10]
```diff
@@ -1,252 +1,346 @@
...
                         Сливочное с апельсиновым джемом и цитрусовой стружкой
                     </p>
                 </div>
 
-                <div class="popular-item-2">
-                    <div class="popular-item-2-hit">
-                        <img class="popular-item-hit" src="img/popular-item-hit.png" alt="Хит">
-                        <img class="popular-item-ice-cream-img" src="img/popular-item-ice-cream-2.jpg" alt="Мороженое">
-                        <p class="popular-item-price">
+                <div class="main__popular-items">
```
Comment (mihail910) [inline index.html]: We need this class once time.

popular-items
-     popular-iteM
-     popular-iteM
-     popular-iteM

File: index.html [hunk 11]
```diff
@@ -1,252 +1,346 @@
...
-                    Для приготовления мороженого используются настоящие сливки и молоко высочайшего качества. Все дополнительные ингредиенты и добавки произведены из натурального, экологически чистого сырья.
-                </p>
-                <p class="about-company-paragraph-3">
-                    Закупка ингредиентов производится только у проверенных фермерских хозяйств и компаний, с которыми нас связывает долговременное сотрудничество.
-                </p>
-                <p class="about-company-paragraph-4">
-                    Доставка нашего мороженого до заказчиков осуществляется в специальном термопаке, который не дает мороженому растаять в пути и позволяет сохранитьпревосходный вкус.
-                </p>
-
+                <div class="advantages__items">
```
Comment (mihail910) [inline index.html]: Look at previous and this section. Right here You use advantages__items and inner - advantage__item. - that's good!

File: index.html [hunk 12]
```diff
@@ -1,252 +1,346 @@
...
-                <form class="index-distribution-form" method="post" action="https://echo.htmlacademy.ru">
-                    <label class="visually-hidden" for="index-distribution-email">
-                        Электронная почта
-                    </label>
-                    <input id="index-distribution-email" type="email" name="email" value="" placeholder="Электронная почта">
-                    <button class="index-distribution-btn" type="submit">
-                        Отправить
-                    </button>
-                </form>
+            <section class="main__promo">
```
Comment (mihail910) [inline index.html]: simple promo

File: index.html [hunk 13]
```diff
@@ -1,252 +1,346 @@
...
+                <article class="main__blog">
```
Comment (mihail910) [inline index.html]: simple blog

File: index.html [hunk 14]
```diff
@@ -1,252 +1,346 @@
...
+                    <h2 class="visually-hidden">Блог</h2>
+                    <p class="main__blog-title">Новое в нашем блоге</p>
+                    <a class="main__blog-link" href="#">10 способов сервировки фруктовых щербетов к столу</a>
+                    <img class="main__blog-picture" src="img/fruit-sherbet.jpg" alt="Фруктовый щербет">
+                </article>
+                <div class="main__distribution">
```
Comment (mihail910) [inline index.html]: simple distribution

File: index.html [hunk 15]
```diff
@@ -1,252 +1,346 @@
...
+                    <p class="main__distribution-description">
+                        Подпишитесь на нашу сладкую рассылку и будете всегда в курсе всего самого вкусного, что у нас происходит. Обещаем не спамить и не слать всякой ненужной ерунды. Честно =)
+                    </p>
+                    <form class="main__distribution-form" method="post" action="https://echo.htmlacademy.ru">
+                        <label for="distribution-email"></label>
+                        <input id="distribution-email" type="email" name="email" value="" placeholder="Электронная почта">
+                        <button class="main__distribution-btn" type="submit">
```
Comment (mihail910) [inline index.html]: we can get rid of type="submit". because button has type submit bt default

File: index.html [hunk 16]
```diff
@@ -1,252 +1,346 @@
...
+                <a class="footer__production-info-links" href="#">Для поставщиков</a>
+                <a class="footer__production-info-links" href="#">О производстве</a>
+                <a class="footer__production-info-links" href="#">Наши документы</a>
+                <a class="footer__production-info-links" href="#">Экологические стандарты</a>
             </div>
-            <div class="footer-links-right">
-                <a class="footer-links" href="#">Наши документы</a>
-                <a class="footer-links" href="#">Экологические стандарты</a>
+            <div class="footer__copyright">
+                <a class="footer__copyright-picture" href="#"><img src="img/htmacademy-logo.png" alt="HTML Academy"></a>
```
Comment (mihail910) [inline index.html]: divide <a> and . Emprty line. It's easier to read.

File: index.html [hunk 17]
```diff
@@ -1,252 +1,346 @@
...
+            <button class="feedback__btn-close" type="button">Закрыть</button>
+            <div class="feedback__item" style="display: none">
+                <form class="feedback__form" id="feedback-form" action="https://echo.htmlacademy.ru" method="post">
+                    <label for="user-name"></label>
+                    <input class="feedback__form-name" id="user-name" type="text" name="user-name" placeholder="Как вас зовут?">
+                    <label for="user-email"></label>
+                    <input class="feedback__form-email" id="user-email" type="email" name="user-email" placeholder="Ваша почта для ответа?">
+                    <label for="user-message"></label>
+                    <textarea class="feedback__form-description" id="user-message" name="user-message" placeholder="Напишите что-нибудь..."></textarea>
+                    <input class="feedback__form-btn" type="submit" value="">
```
Comment (mihail910) [inline index.html]: You should add button

File: index.html [hunk 18]
```diff
@@ -1,12 +1,384 @@
...
+                            Товар№1
+                        </div>
+                        <div class="dropped__basket-item">
+                            Товар№2
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="header__item">
```
Comment (mihail910) [inline index.html]: this container should be placed in the last item of header__item

File: index.html [hunk 19]
```diff
@@ -1,12 +1,384 @@
...
+                            </a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+
+            <div class="header__item">
+                <div class="header__search">
+                    <img class="header__login-picture" src="img/loop.svg" alt="Поиск">
+                    <div class="dropped__search" style="display: none">
```
Comment (mihail910) [inline index.html]: we dont need this wrapper, because we have FORM. form should be styled and be hidden.

File: index.html [hunk 20]
```diff
@@ -1,12 +1,384 @@
...
+                        <form class="header__search-dropped" method="get" action="https://echo.htmlacademy.ru">
+                            <label for="header__search-field"></label>
```
Comment (mihail910) [inline index.html]: should have attribute for="Id"

File: index.html [hunk 21]
```diff
@@ -1,12 +1,384 @@
...
+                            <input id="header__search-field" type="text" name="header__search-field" value="" placeholder="Что ищем?">
+                            <button class="header__search-btn" style="display: none"></button>
+                        </form>
+                    </div>
+                </div>
+                <div class="header__login">
+                    <img class="header__login-picture" src="img/entrance.svg" alt="Вход">
+                    <p class="header__login-description">Вход</p>
+                    <div class="dropped__login" style="display: none">
```
Comment (mihail910) [inline index.html]: we dont need this wrapper, because we have form. Form should be styled and be hidden

File: index.html [hunk 22]
```diff
@@ -1,12 +1,384 @@
...
+                            </div>
+                        </form>
+                    </div>
+                </div>
+                <div class="header__basket">
+                    <a class="header__basket-link" href="#">
+                        <img class="header__login-picture" src="img/basket-empty.svg" alt="Корзина">
+                        <p class="header__login-description">Пусто</p>
+                    </a>
+                    <div class="dropped__basket" style="display: none">
```
Comment (mihail910) [inline index.html]: here we need this wrapper. good. let it be here!

File: index.html [hunk 23]
```diff
@@ -1,12 +1,384 @@
...
+                    <a class="promo-btn" href="#">
+                        Хочу шоколадки!
+                    </a>
+                </div>
+            </section>
+
+            <section class="popular-items">
+                <h2 class="visually-hidden">Популярные товары</h2>
+                
+                <div class="popular-item">
```
Comment (mihail910) [inline index.html]: maybe popular__item ?))

File: index.html [hunk 24]
```diff
@@ -1,12 +1,384 @@
...
+                        </p>
+                    </div>
+                    <p class="popular-description">
+                        Сливочное с апельсиновым джемом и цитрусовой стружкой
+                    </p>
+                </div>
+
+                <div class="popular__item">
+                    <div class="popular__item-box">
+                        <img class="popular-picture-hit" src="img/popular-item-hit.png" alt="Хит">
```
Comment (mihail910) [inline index.html]: replace this image UNDER ice-cream image

File: index.html [hunk 25]
```diff
@@ -1,12 +1,384 @@
...
+            </section>
+
+            <section class="advantages">
+                <h2 class="visually-hidden">О компании</h2>
+                <h3 class="advantages__title">
+                    Магазин Глэйси - это онлайн- и офлайн-магазин по продаже мороженого собственного производства на развес
+                </h3>
+                <div class="advantages__items">
+                    <div class="advantages__item">
+                        <img class="advantages-picture" src="img/ice.png" alt="Мороженое">
```
Comment (mihail910) [inline index.html]: advantages__item-picture

File: index.html [hunk 26]
```diff
@@ -1,12 +1,384 @@
...
+                        <p class="advantages-description">
```
Comment (mihail910) [inline index.html]: advantages__item-description

File: index.html [hunk 27]
```diff
@@ -1,12 +1,384 @@
...
+                            Доставка нашего мороженого до заказчиков осуществляется в специальном термопаке, который не дает мороженому растаять в пути и позволяет сохранитьпревосходный вкус.
+                        </p>
+                    </div>
+                </div>
+            </section>
+
+            <section class="main__container">
+                <article class="blog">
+                    <h2 class="visually-hidden">Блог</h2>
+                    <p class="blog-title">Новое в нашем блоге</p>
```
Comment (mihail910) [inline index.html]: blog__title

File: index.html [hunk 28]
```diff
@@ -1,12 +1,384 @@
...
+                    <a class="blog-link" href="#">10 способов сервировки фруктовых щербетов к столу</a>
```
Comment (mihail910) [inline index.html]: blog__link

File: index.html [hunk 29]
```diff
@@ -1,12 +1,384 @@
...
+                    <img class="blog-picture" src="img/fruit-sherbet.jpg" alt="Фруктовый щербет">
```
Comment (mihail910) [inline index.html]: blog__picture

File: index.html [hunk 30]
```diff
@@ -1,12 +1,384 @@
...
+                        <input id="distribution-email" type="email" name="email" value="" placeholder="Электронная почта">
+                        <button class="distribution-btn">
+                            Отправить
+                        </button>
+                    </form>
+                </div>
+            </section>
+
+            <section class="main__contacts">
+                <div class="main__contacts-item">
```
Comment (mihail910) [inline index.html]: we dont' need this wrapper.

Comment (mihail910) [reply index.html]: and it's contacts. not main__contacts

File: index.html [hunk 31]
```diff
@@ -1,12 +1,384 @@
...
+                    <p class="main__contacts-address">
+                        Адрес главного офиса<br> и офлайн-магазина:
+                    </p>
+                    <p class="main__contacts-address-strong"><b>ул. Большая Конюшенная</b></p>
```
Comment (mihail910) [inline index.html]: we dont'need modifier strong, because we use tag < b >

File: index.html [hunk 32]
```diff
@@ -1,12 +1,384 @@
...
+                    <p class="main__contacts-address-strong">19/8, Санкт-Петербург</p>
```
Comment (mihail910) [inline index.html]: as right above

File: index.html [hunk 33]
```diff
@@ -1,12 +1,384 @@
...
+                    <p class="main__contacts-address">
+                        Для заказов по телефону:
+                    </p>
+                    <a class="main__contacts-link" href="tel:88124502525">
+                        8 812 450-25-25
+                    </a>
+                    <p class="main__contacts-address">
```
Comment (mihail910) [inline index.html]: are use sure that it is address?

File: index.html [hunk 34]
```diff
@@ -1,12 +1,384 @@
...
+                    </a>
+                </div>
+            </section>
+
+        </main>
+
+        <footer class="footer">
+            <div class="footer__social">
+                <a class="footer-social-links" href="#">
+                    <img class="footer-social-picture" src="img/twitter.png" alt="Twitter">
```
Comment (mihail910) [inline index.html]: remove this images. we will do it another way. we would style < a > with background image.

File: index.html [hunk 35]
```diff
@@ -1,12 +1,384 @@
...
+                        Форма обратной связи
+                    </a>
+                </div>
+            </section>
+
+        </main>
+
+        <footer class="footer">
+            <div class="footer__social">
+                <a class="footer-social-links" href="#">
```
Comment (mihail910) [inline index.html]: footer__social-link

Comment (mihail910) [reply index.html]: add attribute target="_blank"

File: index.html [hunk 36]
```diff
@@ -1,12 +1,384 @@
...
+                </a>
+                <a class="footer-social-links" href="#">
+                    <img class="footer-social-picture" src="img/facebook.png" alt="Facebook">
+                </a>
+                <a class="footer-social-links" href="#">
+                    <img class="footer-social-picture" src="img/vk.png" alt="В Контакте">
+                </a>
+            </div>
+            <div class="footer__production-info">
+                <a class="footer__production-info-links" href="#">Для поставщиков</a>
```
Comment (mihail910) [inline index.html]: it is linK. linkS we can use for wrapper.

File: index.html [hunk 37]
```diff
@@ -1,12 +1,384 @@
...
+                <a class="footer__production-info-links" href="#">О производстве</a>
+                <a class="footer__production-info-links" href="#">Наши документы</a>
+                <a class="footer__production-info-links" href="#">Экологические стандарты</a>
+            </div>
+            <div class="footer__copyright">
+                <a class="footer__copyright-picture" href="#">
+                    <img src="img/htmacademy-logo.png" alt="HTML Academy">
```
Comment (mihail910) [inline index.html]: as a wrote about social icons - remove this image. or you can simply make it hidden. display: none

File: index.html (hunk 20 above)
Comment (TsoriTheBoss) [reply index.html]: it's right.
for="header__search-field" and id="header__search-field"
it's label for email field

File: index.html (hunk 33 above)
Comment (TsoriTheBoss) [reply index.html]: i think it's can be address - this text have a address, phone and work time

File: index.html [hunk 38]
```diff
@@ -334,29 +330,19 @@ <h2 class="visually-hidden">Блог</h2>
...
-                <a class="footer-social-links" href="#">
-                    <img class="footer-social-picture" src="img/instagram.png" alt="Instagram">
-                </a>
-                <a class="footer-social-links" href="#">
-                    <img class="footer-social-picture" src="img/facebook.png" alt="Facebook">
-                </a>
-                <a class="footer-social-links" href="#">
-                    <img class="footer-social-picture" src="img/vk.png" alt="В Контакте">
-                </a>
+                <a class="footer__social-link" href="#" target="_blank"></a>
```
Comment (mihail910) [inline index.html]: add modifier like footer__social-link-tw
 footer__social-link-inst
 footer__social-link-fb
 footer__social-link-vk

File: catalog.html [hunk 39]
```diff
@@ -0,0 +1,522 @@
...
+                        <div class="dropped__basket-item">
+                            Товар№1
+                        </div>
+                        <div class="dropped__basket-item">
+                            Товар№2
+                        </div>
+                    </div>
+                </div>
+                <div class="header__item-mode">
+                    <p class="header__item-work-time">C 10 до 20, ежедневно</p>
```
Comment (mihail910) [inline catalog.html]: header__item-mode-time, maybe

File: catalog.html [hunk 40]
```diff
@@ -0,0 +1,522 @@
...
+                    <a class="header__item-phone" href="tel:88124502525">8 812 450-25-25</a>
```
Comment (mihail910) [inline catalog.html]: header__item-mode-phone

File: catalog.html [hunk 41]
```diff
@@ -0,0 +1,522 @@
...
+        <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+
+    <body>
+        <h1 class="visually-hidden">Магазин мороженого</h1>
+        <header class="header">
+            <div class="header__item">
+                <a class="header__logo">
```
Comment (mihail910) [inline catalog.html]: maybe...It is header__logo-link

File: catalog.html [hunk 42]
```diff
@@ -0,0 +1,522 @@
...
+                    <img src="img/index-logo.svg" alt="Глейси">
```
Comment (mihail910) [inline catalog.html]: add width, height attributes

File: catalog.html [hunk 43]
```diff
@@ -0,0 +1,522 @@
...
+                </a>
+                <nav class="header__navigation">
+                    <ul class="header__navigation-items">
+                        <li class="header__navigation-item">
+                            <a class="header__navigation-link" href="#">
+                                Каталог
+                            </a>
+                            <div class="dropped__catalog" style="display: none">
```
Comment (mihail910) [inline catalog.html]: we dont need this wrapper. we should hide <ul> and show it on hover header__navigation-link

File: catalog.html [hunk 44]
```diff
@@ -0,0 +1,522 @@
...
+                                <ul class="header__navigation-dropped">
+                                    <li class="header__navigation-dropped-links">
+                                        <a class="header__navigation-dropped-link-new" href="#">
```
Comment (mihail910) [inline catalog.html]: Why this class differs from links' classes?  See next items links

File: catalog.html [hunk 45]
```diff
@@ -0,0 +1,522 @@
...
+                                О компании
+                            </a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+
+            <div class="header__item">
+                <div class="header__search">
+                    <img class="header__login-picture" src="img/loop.svg" alt="Поиск">
```
Comment (mihail910) [inline catalog.html]: I think it is not login-picture. I believe it's search-picture

File: catalog.html [hunk 46]
```diff
@@ -0,0 +1,522 @@
...
+                            Главная
+                        </a>
+                    </li>
+                    <li class="breadcrumbs__item">
+                        <a class="breadcrumbs__link" href="#">
+                            Каталог
+                        </a>
+                    </li>
+                    <li class="breadcrumbs__item">
+                        Сливочное
```
Comment (mihail910) [inline catalog.html]: Link is lost?

File: catalog.html [hunk 47]
```diff
@@ -0,0 +1,522 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+    <head>
+        <meta charset="utf-8">
+        <title>HTML Academy: Глейси</title>
```
Comment (mihail910) [inline catalog.html]: You can add 'Каталог' :-)

File: index.html [hunk 48]
```diff
@@ -1,12 +1,367 @@
...
+        <link rel="stylesheet" href="css/style.css">
+    </head>
 
-  </body>
+    <body>
+        <h1 class="visually-hidden">Магазин мороженого</h1>
+        <header class="header">
+
+            <div class="header__item">
+                <a class="header__logo">
```
Comment (mihail910) [inline index.html]: header__logo-link?

File: catalog.html (hunk 39 above)
Comment (TsoriTheBoss) [reply catalog.html]: Fixed

File: catalog.html (hunk 41 above)
Comment (TsoriTheBoss) [reply catalog.html]: Fixed

File: catalog.html (hunk 42 above)
Comment (TsoriTheBoss) [reply catalog.html]: Fixed

File: catalog.html (hunk 43 above)
Comment (TsoriTheBoss) [reply catalog.html]: Fixed

File: catalog.html (hunk 44 above)
Comment (TsoriTheBoss) [reply catalog.html]: We need add border-bottom only this link. other links don't have border.

File: catalog.html (hunk 46 above)
Comment (TsoriTheBoss) [reply catalog.html]: Fixed

File: catalog.html (hunk 45 above)
Comment (TsoriTheBoss) [reply catalog.html]: Fixed

File: catalog.html (hunk 47 above)
Comment (TsoriTheBoss) [reply catalog.html]: Fixed

File: index.html (hunk 48 above)
Comment (TsoriTheBoss) [reply index.html]: Fixed

**Pull request merged**